cmake/BLTMacros.cmake

```diff
@@ -161,12 +162,17 @@ endmacro(blt_add_target_link_flags)
 ## the DEPENDS_ON in your blt_add_executable call and it will add the INCLUDES
 ## and LIBRARIES to that executable.
 ##
+## TREAT_INCLUDES_AS_SYSTEM marks the utilizes the compiler's system include path 
+## setting for this libraries include directories.  This is useful if the headers
+## generate warnings you want to not have them reported in your build.
+##
 ## This does not actually build the library.  This is strictly to ease use after
 ## discovering it on your system or building it yourself inside your project.
 ##
 ## Output variables (name = "foo"):
 ##  BLT_FOO_DEPENDS_ON
 ##  BLT_FOO_INCLUDES
+##  BLT_FOO_TREAT_INCLUDES_AS_SYSTEM
```

Comment:
Should the documentation reflect the fact that this one is only optionally generated?

Reply:
im debating whether we should even expose users to what is going on in the background.